crate/operator/restore_backup.py

```diff
@@ -525,7 +525,8 @@ async def _create_backup_repository(
                         await cursor.execute(
                             f"CREATE REPOSITORY {repository_ident} type s3 with "
                             "(access_key = %s, secret_key = %s, bucket = %s, "
-                            "base_path = %s, readonly=true)",
+                            "base_path = %s, readonly=true),",
+                            "max_restore_bytes_per_sec = '120mb'",
```

Comment:
This looks wrong - I think you need to put it within the parentheses? Let's test it on dev also ;)